oauth2client/flask_util.py

```diff
@@ -15,10 +15,17 @@
 """Utilities for the Flask web framework
 
 Provides a Flask extension that makes using OAuth2 web server flow easier.
-The extension includes views that handle the entire auth flow and a @required
-decorator to automatically ensure that user credentials are available.
+The extension includes views that handle the entire auth flow and a
+``@required`` decorator to automatically ensure that user credentials are
+available.
 
-To configure::
+
+Configuration
+=============
+
+To configure, you'll need a set of OAuth2 client ID from the
+`Google Developer's Console <https://console.developers.google.com/project/_/\
```

Comment:
My emacs config makes me overly sensitive to lines longer than 80, so it's great to see this :+1: